api/v1beta1/database_webhook.go

```diff
@@ -58,9 +57,16 @@ var _ webhook.Validator = &Database{}
 func (r *Database) ValidateCreate() error {
 	databaselog.Info("validate create", "name", r.Name)
 
-	err := ValidateSecretTemplates(r.Spec.SecretsTemplates)
-	if err != nil {
-		return err
+	if r.Spec.SecretsTemplates != nil && r.Spec.Templates != nil {
+		return errors.New("using both: secretsTemplates and templates, is not allowed")
+	}
+
+	if r.Spec.SecretsTemplates != nil {
+		fmt.Printf(`secretsTemplates are deprecated, it will be removed in the next API version. Please, consider switching to templates`)
```

Comment:
should we not use log. for output?

Reply:
I'm not sure,it's mostly for user to see in the terminal, so it seems that log are less readable that a simple fmt message.  
But maybe logs are better, I'll check what it looks like

Reply:
I've switched to the webhook logger. I couldn't find out how to print warnings to kubectl output, but I guess we have enough of indicators that secretsTemplates are deprecated. And since we're going to migrate them with the mutating webhook, I think it's fine